buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -43,6 +47,23 @@
     private final PoolChunkList<T> q075;
     private final PoolChunkList<T> q100;
 
+    private final List<PoolChunkListMetric> chunkListMetrics;
+
+    // Metrics for allocations and deallocations
+    private long allocations;
```

Comment:
Is there a reason to keep `allocations` and `deallocations`?  Given that `numAllocations()` and `numDeallocations()` will be called infrequently, we can just do `allocationsTiny + allocationsSmall + allocationsNormal`? Then we can save one increment operations for every (de)allocation.